views/js/qtiCreator/widgets/choices/helpers/formElement.js

```diff
@@ -8,9 +8,21 @@ define(['jquery'], function($) {
                 interaction = choice.getInteraction(),
                 $shuffleToggle = $container.find('[data-role="shuffle-pin"]');
 
+            var _toggleVisibility = function(show) {
```

Comment:
afaik we should not use anymore underscore in function/method names (pls check with @krampstudio )

Reply:
Renamed.